tests/python/conftest.py

```diff
@@ -0,0 +1,93 @@
+import pytest
+import subprocess
+import os
+import time
+import requests
+import signal
+
+import unitycatalog
+
```

Comment:
please add docs on what is in this file. for example, I am confused, this does not have any test in it, only the utility functionalities for pytest to use .. right? if so, why is named conftest.py? is that a standard in python world?

Reply:
oh wow. totally python noob i am :)